utils/swift_build_support/swift_build_support/products/tensorflow.py

```diff
@@ -64,6 +65,11 @@ def build(self, host_target):
         target = ''
         if host_target.startswith('macosx'):
             target = '-DCMAKE_Swift_COMPILER_TARGET=x86_64-apple-macosx10.13'
+        if self.args.enable_x10:
+            x10_inc = os.path.join(self.install_toolchain_path(), 'usr', 'lib',
+                                   'swift', 'x10', 'include')
+        else:
+            x10_inc = ''
```

Comment:
How about hoisting this above the `if self.args.enable_x10`?

Reply:
Done.